bootstrap.js

```diff
@@ -9,25 +9,31 @@ const ResponseBuilder = require('./app/ResponseBuilder');
 module.exports = app => {
     app.get('/*', (req, res) => {
         const responseBuilder = new ResponseBuilder(res);
-        
+
         const requestedUrl = req.url.slice(1);
         const corsBaseUrl = '//' + req.get('host');
-        
+
         console.info(req.protocol + '://' + req.get('host') + req.url);
-        
+
         if(requestedUrl == ''){
             res.send(index);
             return;
         }
 
+        let headers = Object.assign({}, req.headers);
+
+        // This prevents "Error [ERR_TLS_CERT_ALTNAME_INVALID]: Hostname/IP does not match certificate's altnames: Host: localhost"
+        delete headers.host;
```

Comment:
Could you write some unit tests for this functionality please?

Reply:
That would make a lot of sense.